src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -327,26 +333,34 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
             // reset index_w, index_w * dataSize done when built to avoid redundent compute
             mov(reg_index, reg_index_w);
 
+            int step = vlen / (is_i8 ? sizeof(int8_t) : sizeof(float));
+
             Xbyak::Label nn_loop_label;
             Xbyak::Label nn_loop_end_label;
             Xbyak::Label nn_tail_loop_label;
             Xbyak::Label nn_tail_loop_end_label;
 
             L(nn_loop_label);   // inner loop
             {
-                cmp(reg_work_amount, vector_step);
+                cmp(reg_work_amount, step);
                 jl(nn_loop_end_label, T_NEAR);
+                if (is_i8) {
+                    gather_i8_i32_indices_store(reg_dst, reg_src_h, reg_index, false);
+                    if (attr_.post_ops_.len() != 0) {
+                        IE_THROW() << "Interpolate I8 doesn't support fusing";
+                    }
```

Comment:
Report exceptions will just lead some models fail to run, we need manually handle it to run successfully. Two options:

1. Disable fusion directly for this case in [canBeFuse()](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/interpolate.cpp#L3228). Then there will be no post-ops in this case, and post-ops is processed in another node successfully.
2. Support fusion in all cases. If there are post-ops, we need gather u8/i8(src_prc) data to FP32 in register(need precision convert), apply post-ops fusion, store FP32 to dst_prc with store_num. In this case, vector step is always (vlen / sizeof(float));

@dmitry-gorokhov , @ceciliapeng2011 what do you think which one is preferred?

Reply:
For now, added a check to `Interpolate::canFuse()`